biochatter/vectorstore_host.py

```diff
@@ -1,196 +1,541 @@
-import logging
+import logging, uuid, random
 from typing import List, Optional, Tuple, Dict
-import re
-import base64
 from pymilvus import (
     MilvusException,
     connections,
     Collection,
     utility,
+    DataType,
     FieldSchema,
+    CollectionSchema,
 )
 from langchain.vectorstores import Milvus
 from langchain.embeddings import OpenAIEmbeddings
 from langchain.schema import Document
```

Comment:
@fengsh27 is this module meant to completely replace the `vectorstore.py` module, i.e. the `DocumentEmbedder` and `DocumentReader` classes?

Reply:
No, the intention of this module is not to replace `DocumentEmbedder`. From my perspective, `VectorDatabaseHostMilvus` functions as milvus driver, while the `DocumentEmbedder` class serves as the primary interface exposed to ChatGSE.  
The `DocumentEmbedder` takes charge of text splitting and delegates database operations to `VectorDatabaseHostMilvus`.  In the future, we may need to introduce other vector database drivers like `VectorDatabaseHostChroma`. It will be `DocumentEmbedder` to manage various vector databases.

Reply:
Indeed, the `test_document_summariser` comprehensively covers all facets of the `VectorDatabaseHost` class, encompassing crucial functionalities such as `store_embedding`, `similarity_search`, `get_all_documents`, and `remove_document`.